infrastructure/crypto.ts

```diff
@@ -15,7 +15,7 @@
 import * as forge from 'node-forge';
 
 // Keys are in OpenSSH format
-export class KeyPair {
```

Comment:
Revert. This is uses as a struct, not interfaces. I see interfaces with fields as an anti-pattern, since it forces implementation.